code/modules/paperwork/paper.dm

```diff
@@ -344,7 +344,7 @@
 		var/const/station_text = "\[Station name\]"
 		var/list/menu_list = list() //text items in the menu
 		menu_list.Add(usr.real_name) //the real name of the character, even if it is hidden
-		if(usr.real_name != usr.name || usr.name != "unknown") //if the player is masked or the name is different a new answer option is added
```

Comment:
I don't think you meant to include this change. If you did, please add it to your PR description and changelog

Reply:
It's just removing unnecessary validation.
Since the real name was added in any case duplicating itself in the list, maybe it had some other purpose than duplication.